Tests/IntegrationTests/UIKit/Views/SupportedPaymentMethodLogosViewTests.swift

```diff
@@ -0,0 +1,41 @@
+//
+// Copyright (c) 2024 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+@_spi(AdyenInternal) @testable import Adyen
+import XCTest
+
+class SupportedPaymentMethodLogosViewTests: XCTestCase {
+    
+    override func run() {
+        AdyenDependencyValues.runTestWithValues {
+            $0.imageLoader = ImageLoaderMock()
+        } perform: {
+            super.run()
+        }
+    }
+    
+    func test_setup() throws {
+        let trailingText = "TRAILING_TEXT"
+        let supportedLogosView = SupportedPaymentMethodLogosView(
+            imageUrls: [URL(string: "https://adyen.com")!],
+            trailingText: trailingText
+        )
+        XCTAssertEqual(supportedLogosView.subviews.count, 0)
+        XCTAssertNil(supportedLogosView.content)
+        
+        let viewController = ADYViewController(view: supportedLogosView)
+        setupRootViewController(viewController) // Providing a window
```

Comment:
Could you thy another approach that I used to convert UI tests to integration tests actually to remove relying on mounting the whole view hierarchy into a real window? This will speed up the test significantly.